app/views/licence/_action.html.erb

```diff
@@ -2,22 +2,26 @@
   <h1>How to <%= action %></h1>
 </header>
 
-<% if @licence_details.authority["actions"][action].size > 1 %>
+<% if @licence_details.authority["actions"][action].size > 1 && @licence_details.any_action_uses_licensify? %>
   <p>There is more than one online form available for this licence:</p>
 
   <ol>
     <% @licence_details.authority["actions"][action].each_with_index do |link, i| %>
-      <li><%= link_to link['description'], "#form-#{i+1}" %></li>
+      <% if link['uses_licensify'] %>
+        <li><%= link_to link['description'], "#form-#{i+1}" %></li>
+      <% end %>
     <% end %>
   </ol>
 <% end %>
 
 <% @licence_details.authority["actions"][action].each_with_index do |link, i| %>
-  <section>
-    <h2><a id="form-<%= i+1 %>"></a><%= i+1 %>. <%= link['description'] %></h2>
+  <% if link['uses_licensify'] %>
+    <section>
+      <h2><a id="form-<%= i+1 %>"></a><%= i+1 %>. <%= link['description'] %></h2>
 
-    <%= simple_format link['introduction'] %>
+      <%= simple_format link['introduction'] %>
 
-    <p><%= link_to "Apply online", link['url'], :class => "button" %></p>
-  </section>
+      <p><%= link_to "Apply online", link['url'], :class => "button" %></p>
+    </section>
+  <% end %>
```

Comment:
I don't know enough about the contents of the api responses to know if this question makes sense, but instead of rendering nothing if a given link doesn't use licensify, could we render the information we do have about the link and put up the "licensify_unavailable" partial for it?

Is it useful for someone to know that there are several links for a given action for a license and some of them can be applied for online via licensify, but others can't?  They're presumably still things that you might need to know about or do, so maybe we should provide the user that info even if they can't apply online?

Reply:
Done